build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator/docstring.rb

```diff
@@ -140,6 +140,9 @@ def html_to_markdown(html, options = {})
           # <important> tag doesn't render well
           html = html.gsub(/<important>(.+?)<\/important>/){ "<p>#{$1}</p>" }
 
+          # same with <replaceable>, sub it out for italic
```

Comment:
this requires context of previous comment (which could be changed), just put <replaceable> tag doesnt render well